src/frontend/komponenter/Fagsak/Behandlingsmeny/Behandlingsmeny.tsx

```diff
@@ -0,0 +1,98 @@
+import { MenuElipsisHorizontalIcon } from '@navikt/aksel-icons';
+import { Button, Dropdown } from '@navikt/ds-react';
+import * as React from 'react';
+
+import { EndreBehandlendeEnhet } from './EndreBehandlendeEnhet/EndreBehandlendeEnhet';
+import { GjennoptaBehandling } from './GjennoptaBehandling/GjennoptaBehandling';
+import { HenleggBehandling } from './HenleggBehandling/HenleggBehandling';
+import { HentOppdatertKravgrunnlag } from './hentOppdatertKravgrunnlag/HentOppdatertKravgrunnlag';
+import { HistoriskeVurderinger } from './HistoriskeVurderinger/HistoriskeVurderinger';
+import { LeggTilFjernBrevmottakere } from './LeggTilFjernBrevmottakere/LeggTilFjernBrevmottakere';
+import { OpprettBehandling } from './OpprettBehandling/OpprettBehandling';
+import { SettBehandlingPåVent } from './SettBehandlingPåVent/SettBehandlingPåVent';
+import { SettBehandlingTilbakeTilFakta } from './SettBehandlingTilbakeTilFakta/SettBehandlingTilbakeTilFakta';
+import { useApp } from '../../../context/AppContext';
+import { useBehandling } from '../../../context/BehandlingContext';
+import { ToggleName } from '../../../context/toggles';
+import { useToggles } from '../../../context/TogglesContext';
+import { Fagsystem } from '../../../kodeverk';
+import { useFagsakStore } from '../../../stores/fagsakStore';
+import { Behandlingssteg, Behandlingstatus } from '../../../typer/behandling';
+import { RessursStatus } from '../../../typer/ressurs';
+
+export const Behandlingsmeny: React.FC = () => {
+    const { behandling, ventegrunn, erStegBehandlet, aktivtSteg } = useBehandling();
+    const { fagsystem, ytelsestype } = useFagsakStore();
+    const { innloggetSaksbehandler } = useApp();
+    const erProd =
+        !window.location.hostname.includes('dev') &&
```

Comment:
Jeg får en advarsel om at man bør bruke `globalThis` istedenfor `window` da den er mer tilgjengelig for flere miljøer og har Typescript støtte. Sikkert ikke viktig men kanskje verdt å se på: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#type-checking-for-globalthis